docs/src/compatibility.md

```diff
@@ -78,3 +78,12 @@ PkgB = "≥ 1.2.3"  # [1.2.3,  ∞)
 PkgC = "= 1.2.3"  # [1.2.3, 1.2.3]
 PkgD = "< 1.2.3"  # [0.0.0, 1.2.2]
 ```
+
+### Range specifiers
+
+Ranges can also be used to specify version ranges:
+
+```toml
+[compat]
+PkgA = "0.1 - 0.2.3" # [0.1.0,  0.2.3)
```

Comment:
This is not what a range should mean: the second end-point should be inclusive, and any unspecified trailing numbers should be considered to be wildcards. I.e.
* `0.1-0.2.3` means `[0.1, 0.2.3]`
* `0.1-0.2` means `[0.1, 0.3)`
* `0.1-0` means `[0.1, 1.0)`

This seems to be what you've implemented, but it would be good to have some tests and, of course, the comment here should be fixed.

Reply:
Thanks for taking a look. I didn't notice the ) vs the ]. I just copied in pasted from the doc above.

Did you see the tests already in the commit? I am not really familiar with semver, so if you could recommend any additional tests, that would be helpful.